requirements-test.txt

```diff
@@ -2,3 +2,4 @@ coverage
 coveralls
 isort
 flake8
+openwisp-utils>=0.2.1
```

Comment:
in this module openwisp-utils is already a requirement: https://github.com/openwisp/django-netjsonconfig/blob/master/requirements.txt, so we can avoid adding it here because it will be pulled when installing the main requirements